README.md

```diff
@@ -23,7 +23,64 @@ This will create a clone of the project where you can make changes as you desire
 ##### STEP 2: Running the Game
 Type `python main.py` to run the game.
 
-![Alt text](https://github.com/vineetjc/pygame-Search-the-Key/blob/master/screenshots/1.png)
+GAMEPLAY AND SCREENSHOT
+
+(https://github.com/vineetjc/pygame-Search-the-Key/blob/master/screenshots/Capture.png)
+
+(https://github.com/vineetjc/pygame-Search-the-Key/blob/master/screenshots/Capture2.png)
 
+![Alt text](https://github.com/vineetjc/pygame-Search-the-Key/blob/master/screenshots/1.png)
 
 ![Alt text](https://github.com/vineetjc/pygame-Search-the-Key/blob/master/screenshots/2.png)
+
+(https://github.com/vineetjc/pygame-Search-the-Key/blob/master/screenshots/Capture5.png)
+
+DEVELOPING THE GAME
```

Comment:
Let's not have an entire guide on git, check the README on https://github.com/vineetjc/pygame-Coin-Fall- if that gives a better structure?